ui/pages/Project/constants.js

```diff
@@ -504,3 +504,8 @@ export const TAG_FORM_FIELD = {
   parse: value => (value || []).map(name => ({ name })),
   validate: value => (value && value.length ? undefined : 'Required'),
 }
+
+export const PHENOTYPE_PRIORITIZATION_FIELDS = {
+  exomiser: 'Exomiser',
+  lirical: 'LIRICAL',
```

Comment:
I don't think it look good to use different casing strategies for the different tools, so I think we should pick one and be consistent and then you can just apply whatever function directly to the tool instead of needing to define a mapping

Reply:
I added this as an extra change so that the our display for the tool names matches how they're actually capitalized - LIRICAL is all [uppercase](https://lirical.readthedocs.io/en/latest/) and exomiser [isn't](https://www.sanger.ac.uk/tool/exomiser/). I can take this out though as it's not an explicit part of the ticket.

Reply:
removed this change from this PR 😆